cmake/FindDPCPP.cmake

```diff
@@ -38,7 +38,7 @@ find_library(DPCPP_LIB_DIR NAMES sycl sycl6 PATHS "${DPCPP_BIN_DIR}/../lib")
 
 add_library(DPCPP::DPCPP INTERFACE IMPORTED)
 
-set(DPCPP_FLAGS "-fsycl;-mllvm;-enable-global-offset=false;")
```

Comment:
This is needed for the nvidia backend. My understanding is that you will get a warning only if the compiler is not including the nvidia backend, otherwise it will just be ignored.

Could you add it back under an `if("${DPCPP_SYCL_TARGET}" STREQUAL "nvptx64-nvidia-cuda")` check?

Reply:
It is part of the PR on line [54](https://github.com/codeplaysoftware/cutlass-fork/pull/104/files#diff-e86f49d81325b91f324c14f856227de52f1d80b7d900f7185ef06f96bb69dbc4R54) as part of compiler flag. I think there is no harm on putting it inside if in line [48](https://github.com/codeplaysoftware/cutlass-fork/pull/104/files#diff-e86f49d81325b91f324c14f856227de52f1d80b7d900f7185ef06f96bb69dbc4R48)

Reply:
Is this needed for nvidia backend for linking?